File: Iniciante/URI 1005.go
package main
 
import (
    "fmt"
)
 
func main() {
    var a, b, media float64

    fmt.Scanf("%f", &a)
    fmt.Scanf("%f", &b)
    media = ((3.5 * a) + (7.5 * b)) / 11
    fmt.Printf("MEDIA = %.5f\n", media)
}

File: Iniciante/URI 1005.c
#include <stdio.h>
 
int main() { 
    float a, b, media;

    scanf("%f %f", &a, &b);
    media = ((3.5 * a) + (7.5 * b)) / 11;
    printf("MEDIA = %.5f\n", media);

    return 0;
}